lxd-provisioning/README.md

```diff
@@ -0,0 +1,187 @@
+### Setting up K8s Cluster using LXC/LXD 
+This is not a recommended way to create k8s cluster, but for learning and testing purpose we can use this.
+
+```
+For running LXC, Create a EC2 instance of t2.medium size which will sufficient for running LXC with 3 nodes, each of 2 CPU's and 2Gi of memory.
+```
+#### Installing the LXC on Ubuntu 
+```
+➤ ➤ ADIL ~  sudo apt-get update && apt-get install lxc -y
```

Comment:
```suggestion
$ sudo apt update && apt install lxc -y
```
Similary for all the commands if you could remove the PROMPT and change it to just $, it will look more generic.